src/sdk/namespace_fs.js

```diff
@@ -3612,11 +3612,15 @@ class NamespaceFS {
     }
 
     async _is_storage_class_supported(storage_class) {
+        const glacier_storage_classes = [
```

Comment:
Maybe old school, but I find it weird that we define this constant inside the check function. I would define it in module scope probably...

Reply:
@guymguym I just did it so that I don't have to chain a lot of ifs and could simply use `.include`. There was no other reason for defining that variable 🙂 .